pom.xml

```diff
@@ -54,7 +54,7 @@
     <testng.version>6.8</testng.version>
     <guava.version>17.0</guava.version>
     <ome-common.version>5.3.1</ome-common.version>
-    <ome-model.version>5.3.1</ome-model.version>
+    <ome-model.version>5.3.2-SNAPSHOT</ome-model.version>
```

Comment:
Now ome-model 5.4.0 is out and as per https://github.com/ome/ome-model/commit/cfed4019e445eba49ff7440f967c2e4886be3b98, this should be updated to 6.0.0-SNAPSHOT